examples/evkbmimxrt1060/test/include/kvstore_config.h

```diff
@@ -45,8 +45,8 @@ typedef enum KVStoreKey
 } KVStoreKey_t;
 
 /* Define default values for common attributes */
-#define THING_NAME_DFLT        democonfigCLIENT_IDENTIFIER

```

Comment:
`THING_NAME_DFLT` and `MQTT_ENDOPOINT_DFLT` configuration seems not used and can be removed